beginner_source/deeplabv3_on_android.rst

```diff
@@ -1,234 +1,10 @@
 Image Segmentation DeepLabV3 on Android
 =================================================
 
-**Author**: `Jeff Tang <https://github.com/jeffxtang>`_
+This tutorial has been deprecated. There is a new tutorial on this topic.
```

Comment:
We should probably say something like: PyTorch Mobile is no longer actively supported. Please check out  ExecuTorch.